data/sql/updates/pending_db_world/rev_1646319575149213791.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1646319575149213791');
+
+DELETE FROM `quest_offer_reward_locale` WHERE  `RewardText` IS NULL;
+DELETE FROM `quest_request_items_locale` WHERE  `CompletionText` IS NULL;
```

Comment:
This is a a bad idea. The ID's in the table are sniffed, the locale texts needs to be filled in. Using this will removed parse data from sniffs.

Reply:
**I don't think it's a bad idea**

Filter on the `quest_offer_reward_locale` and `quest_request_items_locale` tables with value (NULL) so you can see that many frFR language quests are empty. https://github.com/azerothcore/azerothcore-wotlk/issues/10886

Even many quests do not require translations in `quest_request_items_locale` , example quest 10177

https://wotlkdb.com/?quest=10177 
https://es.tbc.wowhead.com/quest=10177/problems-in-auchindoun

I imagine that with the client in that language, those npc should come up blank when delivering those missions.